tools/text_processing/text_processing/grep.xml

```diff
@@ -59,6 +61,7 @@
             <option value="-i">case insensitive</option>
             <option value="">case sensitive</option>
         </param>
+        <param name="only_matching" type="boolean" argument="only-matching" label="Show only nonempty parts of lines that match" truevalue="-o" falsevalue="" checked="false" help="This outputs only the match" />
```

Comment:
`argument` should be `-o` isn't it?

Reply:
either: https://linux.die.net/man/1/grep

Reply:
Than the `--` are missing :)

Reply:
This PR sat so long I deleted the original repo. I won't be able to edit the PR.